### DISCUSSION TITLE
Is this lib still maintained?

### DISCUSSION THREAD

**Discussion by benjaminbours** [ctrlplusb/easy-peasy#762] [Q&A] [answered]
Just the simple question in the title. I would to know in order to base future project on solid bases. Thanks in advance

**Comment by ctrlplusb** [comment]
It is still being used by myself, and will be getting a major update soon.

**Comment by ctrlplusb** [comment, marked as answer]
We have @jmyrland onboard as a core team member now. Him and myself will be working together to get the maintenance rounds done for Easy Peasy. We will hopefully be inviting more core team members too.

This should hopefully alleviate fears that my personal life will affect maintenance rounds. ♥️
